package.json

```diff
@@ -28,8 +28,8 @@
     "test": "jest"
   },
   "main": "dist/vuetable-3.umd.js",
-  "dependencies": {
-    "axios": "1.6.0",
+  "peerDependencies": {
+    "axios": "^1.6.0",
```

Comment:
Does it not need to be at least [Release v1.7.4](https://github.com/axios/axios/releases/tag/v1.7.4) to mitigate the CSV  security issues?

Reply:
To explicitly mitigate it, yes. My goal here was that a user should be able to upgrade to that version on their own and the package should not throw an error. This change was the smallest change I could do to achieve this.

Reply:
Added the version upgrade